py/server/tests/test_pt_proxy.py

```diff
@@ -318,6 +318,40 @@ def test_agg_all_by(self):
             for gct in agg_pt_proxy.target.constituent_tables:
                 self.assertGreaterEqual(gct.size, 1)
 
+    def test_query_scope(self):
```

Comment:
Should the whole set of scope tests from table be here?

Reply:
I don't think it is necessary. The new tests for table cover most if not all the use cases to verify that we build the correct scope for formula evaluation

Reply:
I suggested this because @rcaudy implied that partitioned tables were somehow different than other cases.